DataFormats/L1TrackTrigger/interface/TTStub.h

```diff
@@ -0,0 +1,187 @@
+/*! \class   TTStub
+ *  \brief   Class to store the L1 Track Trigger stubs
+ *  \details After moving from SimDataFormats to DataFormats,
+ *           the template structure of the class was maintained
+ *           in order to accomodate any types other than Phase2TrackerDigis
+ *           in case there is such a need in the future.
+ *
+ *  \author Andrew W. Rose
+ *  \author Nicola Pozzobon
+ *  \date   2013, Jul 12
+ *
+ */
+
+#ifndef L1_TRACK_TRIGGER_STUB_FORMAT_H
+#define L1_TRACK_TRIGGER_STUB_FORMAT_H
+
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/L1TrackTrigger/interface/TTCluster.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+
+template< typename T >
+class TTStub
+{
+  public:
+    /// Constructors
+    TTStub();
+    TTStub( DetId aDetId );
+
+    /// Destructor
+    ~TTStub();
+
+    /// Data members:   getABC( ... )
+    /// Helper methods: findABC( ... )
+
+    /// Clusters composing the Stub
+    std::vector< edm::Ref< edmNew::DetSetVector< TTCluster< T > >, TTCluster< T > > > getClusterRefs() const { return theClusterRefs; }
```

Comment:
A stub is a pair of clusters with some additional information (offset, etc.).
I changed the implementation in an update of this PR, where the cluster refs are kept in a lighter way (no need for a vector).